src/completer/index.css

```diff
@@ -0,0 +1,58 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2016, Jupyter Development Team.
+|
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+
+:root {
+  --jp-private-completer-code-font-size: 14px;
+  --jp-private-completer-item-height: 24px;
+}
+
+
+.jp-Completer {
+  background: white;
+  border: var(--jp-border-width) solid var(--jp-border-color1);
+  list-style-type: none;
+  overflow: auto;
+  padding: 0;
+  margin: 4px 0 0 -4px;
+  max-height: calc((10 * var(--jp-private-completer-item-height)) + (2 * var(--jp-border-width)));
+  min-height: calc(var(--jp-private-completer-item-height) + (2 * var(--jp-border-width)));
+  position: absolute;
+  z-index: 10001;
+}
+
+
+.jp-Completer.jp-mod-outofview {
+  display: none;
+}
+
+
+.jp-Completer-item {
+  margin: 0;
+  min-width: 150px;
+  padding: 0 2px;
+}
+
+
+.jp-Completer-item:hover, .jp-Completer-item.jp-mod-active {
+  background: #BFBFBF;
+}
+
+
+.jp-Completer-item code {
+  font-size: var(--jp-private-completer-code-font-size);
+  height: var(--jp-private-completer-item-height);
+  padding: 0;
+  margin: 0;
+}
+
+
+.jp-Completer-item mark {
+  font-weight: bold;
+  text-decoration: underline;
```

Comment:
I think this makes is overly distracting, and with the completer UI aligned with the stuff the user typed, I don't think it is needed